text/0097-log-data-model.md

```diff
@@ -0,0 +1,995 @@
+value of `Resource`. Can contain for example information about the application
+that emits the record or about the infrastructure where the application runs.
+Data formats that represent this data model may be designed in a manner that
+allows the `Resource` field to be recorded only once per batch of log records
+that come from the same source. SHOULD follow OpenTelemetry
+[semantic conventions for Resources](https://github.com/open-telemetry/opentelemetry-specification/tree/master/specification/resource/semantic_conventions).
+
+### Field: `Attributes`
+
+Type: key/value pair list.
+
+Description: Additional information about the specific event occurrence. "value"
+of each pair is of `any` type. Unlike the `Resource` field, which is fixed for a
+particular source, `Attributes` can vary for each occurrence of the event coming
+from the same source. Can contain information about the request context (other
+than TraceId/SpanId). SHOULD follow OpenTelemetry
+[semantic conventions for Attributes](https://github.com/open-telemetry/opentelemetry-specification/tree/master/specification/trace/semantic_conventions).
+
+## Example Log Records
+
+Below are examples that show one possible representation of log records in JSON.
+These are just examples to help understand the data model. Don’t treat the
+examples as _the_ way to represent this data model in JSON.
+
+This document does not define the actual encoding and format of the log record
+representation. Format definitions will be done in separate OTEPs (e.g the log
+records may be represented as msgpack, JSON, Protocol Buffer messages, etc).
+
+Example 1
+
+```javascript
+{
+  "Timestamp": 1586960586000, // JSON needs to make a decision about 
+                              // how to represent nanoseconds.
+  "Attributes": {
+    "http.status_code": 500,
+    "http.url": "http://example.com",
+    "my.custom.application.tag": "hello",
+  },
+  "Resource": {
+    "service.name": "donut_shop",
+    "service.version": "semver:2.0.0",
+    "k8s.pod.uid": "1138528c-c36e-11e9-a1a7-42010a800198",
+  },
+  "TraceId": "f4dbb3edd765f620", // this is a byte sequence 
+                                  // (hex-encoded in JSON)
+  "SpanId": "43222c2d51a7abe3",
+  "SeverityText": "INFO",
+  "SeverityNumber": 9,
+  "Body": "20200415T072306-0700 INFO I like donuts"
+}
+```
+
+Example 2
+
+```javascript
+{
+  "Timestamp": 1586960586000,
+  ...
+  "Body": {
+    "i": "am",
+    "an": "event",
+    "of": {
+      "some": "complexity"
+    }
+  }
+}
+```
+
+Example 3
+
+```javascript
+{
+   "Timestamp": 1586960586000,
+   "Attributes":{
+      "http.scheme":"https",
+      "http.host":"donut.mycie.com",
+      "http.target":"/order",
+      "http.method":"post",
+      "http.status_code":500,
+      "http.flavor":"1.1",
+      "user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.149 Safari/537.36",
```

Comment:
I suspect that there is a typo on the key being `user_agent`. My understanding of the semanti conventions is that the key should be `http.user_agent`. Am I right?
Ref https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/http.md#common-attributes

Reply:
Yes, good catch, thank you. Fixed.